src/app/components/Root.js

```diff
@@ -110,6 +111,7 @@ class Root extends Component {
                   <Route path=":team/unmatched-media(/:query)" component={UnmatchedMedia} />
                   <Route path=":team/published(/:query)" component={Published} />
                   <Route path=":team/feed/:feedId/edit" component={EditFeed} />
+                  <Route path=":team/feed-team/:feedTeamId" component={EditFeedTeam} />
```

Comment:
@brianfleming does this route seem alright from a product perspective? @dariusk suggested it could be something more meaningful to users

Reply:
@amoedoamorim this is the route for a collaborating workspace (not the creator) to edit (edit what they can) for a  shared feed that they are participating in, correct?

If I am correct, in the future we will most likely allows workspaces to be more self sufficient when it comes to managing shared feeds, we are limited what they can edit right now to reduce complexity. So in reality, I think of this page as more like the route above `:team/feed:feedId/edit`, thats what the user is doing, they can just edit far less than the creator org.

So maybe it would be better if the route was aligned with that concept?
`:team/feed:feedId/collaboration/:feedTeamId` or some other modifying word other than collaboration?

cc @dariusk

Reply:
I think something like `:team/feed/:feedId/collaborate/:feedTeamid` would be great, yeah. I prefer the verb here since it's holding the same place as `edit` normally would.

So to be clear: if a workspace called `primary` invited me to collaborate on a shared feed of id `12`, and I am a member of the workspace called `seconday` that was invited and accepted, then the url to change the list associated with the shared feed would look like

`/secondary/feed/12/collaborate/16`

where `16` was whatever id we got when we were invited by `primary`.

Ultimately I think `slug` for shared feeds is what we really need here because if the shared feed `12` is called "Our Shared Feed" with a slug of `our-shared-feed` then it would look like

`/secondary/feed/our-shared-feed/collaborate/16`

which is imo much much better.

Reply:
Also occurs to me: there can only be one `FeedTeam` edge between a feed and a team, right? Would it be possible to just go to 

`secondary/feed/our-shared-feed/collaborate`

and then essentially query the database, asking "hey is my team associated with `our-shared-feed`? if so please return the `FeedTeam` object". That way we would entirely avoid the ID in the url

Reply:
This last thought is exactly what I was telling @brianfleming:

We could actually have the same `:team/feed/:feedId/edit` pattern for creator and invited team.
We'd just need to make the graphql api return a `feed_team` given a `teamSlug` and a `feed_id`.